docs/serverless/05-02-serverless-chart-config.md

```diff
@@ -14,9 +14,13 @@ To configure the Serverless chart, override the default values of its `values.ya
 
 This table lists the configurable parameters, their descriptions, and default values for both cluster and local installations.
 
+> **NOTE:** Limited memory and CPU resources on Minikube directly affects the Serverless functionality as you can process only a limited number of Functions at the same time, and their processing time is significantly longer. To process large workloads, we recommend using the cluster setup.
```

Comment:
```suggestion
> **NOTE:** Limited memory and CPU resources on Minikube directly affect the Serverless functionality as you can process only a limited number of Functions at the same time and their processing time is significantly longer. To process large workloads, we recommend using the cluster setup.
```

Reply:
And to shorten this sentence, you could split it as: 
`Limited memory and CPU resources on Minikube directly affect the Serverless functionality as you can process only a limited number of Functions at the same time. Also, their processing time is significantly longer.`
or
`Limited memory and CPU resources on Minikube directly affect the Serverless functionality as you can process only a limited number of Functions at the same time. Their processing time is significantly longer too.` or `as well.`

Again, I leave it to your discretion.